### ISSUE TITLE
Handle the `arbitrage_exp` equation family

### ISSUE THREAD

**Issue by spencerlyon2** [EconForge/Dolo.jl#15]

**Comment by jagman88** [comment]
Yep. It's used to write arbitrage equations as functions of expectations functions, which are then themselves defined in an "expectations" section. For example in the RBC model (apologies, I can't see markup on my phone):

arbitrage_exp:
    - 1/c - m
    ...

expectations:
    - m - beta*c(1)*(rk(1) + 1 - delta)
